### PR TITLE
Add parliament

### PR DESCRIPTION
**Pull request by BastianZim**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

_Created with Grayskull_

### CODE REVIEW

**Comment by BastianZim** [comment]
@conda-forge/help-python Would be ready for review, thanks!

File: recipes/parliament/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  license_file: LICENSE
+  description: |
+    parliament is an AWS IAM linting library. It reviews policies looking for problems such as:
+
+    * malformed json
+    * missing required elements
+    * incorrect prefix and action names
+    * incorrect resources or conditions for the actions provided
+    * type mismatches
+    * bad policy patterns
```
Comment (BastianZim) [inline recipes/parliament/meta.yaml:56]: Is that the correct way to declare a list? The output should be like in here: https://github.com/duo-labs/parliament/blob/main/README.md

Comment (chrisburr) [reply recipes/parliament/meta.yaml:56]: This should be fine. It will get rendered as plain text in some places but most treat it as markdown.

Comment (BastianZim) [reply recipes/parliament/meta.yaml:56]: Ok, thanks for the feedback!

**Pull request merged**